src/routes/safe-apps/__tests__/get-safe-apps.e2e-spec.ts

```diff
@@ -41,6 +41,14 @@ describe('Get Safe Apps e2e test', () => {
               name: expect.any(String),
               url: expect.any(String),
               chainIds: expect.arrayContaining([chainId]),
+              iconUrl: expect.any(String),
+              description: expect.any(String),
+              accessControl: expect.objectContaining({
+                type: expect.any(String),
+              }),
+              tags: expect.any(Array),
+              features: expect.any(Array),
+              socialProfiles: expect.any(Array),
```

Comment:
Regarding the assertions above: why not use `.expect((response) => { ... })`? `then` is building on the promise API but I think we can leverage the test api provided by `supertest` to test the body.

```typescript
.expect((response) => {
  expect(response.body).toBeInstanceOf(Array);
  ...
});
```

We are also not using the map result on the body. A `forEach` seems to be a better fit for this validation as the `expect` is just a side effect.

Reply:
Yes, I agree it's simpler that way. I've changed it in  [62c5973](https://github.com/safe-global/safe-client-gateway-nest/pull/383/commits/62c597389202d33c6d35481e3eebfb43b6f7d392)